backend/img_upload.ts

```diff
@@ -0,0 +1,45 @@
+import dotenv from 'dotenv'
+import axios from 'axios'
+import FormData from 'form-data'
+import fs from 'fs'
+
+// initialize configuration
+dotenv.config()
+
+const PINATA_JWT = process.env.PINATA_JWT ?? ''
+const PINATA_PINFILE_URL = process.env.PINATA_PINFILE_URL ?? ''
+
+const JWT = `Bearer ${PINATA_JWT}`
+
+const pinFileToIPFS = async (name: string) => {
+  const formData = new FormData()
+  const IMG_PATH = './backend/me.jpg'
```

Comment:
Same here. The hard coded `./backend/me.jpg` is in your `.gitignore` file. 
So, if someone clones your repo and run this code, it'd break. Let's fix this.